sheetsAuth.js

```diff
@@ -83,6 +83,28 @@ function formatSheetResults(auth) {
   })
 }
 
+/* for getting datas from zoom raw data output sheet */
+function getAttendanceDatas(auth) {
+  return new Promise( (resolve, reject) => {
+    const sheets = google.sheets({version: 'v4', auth});
+    sheets.spreadsheets.values.get({
+      spreadsheetId: process.env.RPT_ATTENDANCE_OUTPUT,
+      range: 'Raw Data!A1:AA3', // A1:AA3 is a test range
```

Comment:
Can you used Named Ranges with their API?  It's less brittle that explicit ranges.  I don't know if it's supported though.